features/coach_can_create_session.feature

```diff
@@ -0,0 +1,31 @@
+@javascript
+Feature: Coach can create session
+
+    As a coach
+    In order to utilize my spare time
+    I would like to be able to make sessions on Tribal
+
+    Background:
+        Given the following price table exists
+            | price_point | trainees |
+            | low         | 4        |
+            | medium      | 4        |
+            | high        | 4        |
+
+        And the following user exists
+            | email           | password | password_confirmation | first_name | last_name |
+            | coach@email.com | password | password              | Coach-Jon  | Doe       |
+        And I visit the site
+        And I am logged in as 'coach@email.com'
+        And I click 'Add Session'
+
+    Scenario: Coach can create session
+        Given I fill in 'session[title]' field with 'Spinning'
+        And I fill in 'session[price_point]' field with '1'
```

Comment:
Can we use 'Low,medium and high' for price points instead of numbers?

Reply:
We need to refactor our price table to change this, we have created a chore for this in our PT.